imapclient/imapclient.py

```diff
@@ -472,6 +476,22 @@ def select_folder(self, folder, readonly=False):
         self._command_and_check('select', self._normalise_folder(folder), readonly)
         return self._process_select_response(self._imap.untagged_responses)
 
+    def unselect_folder(self):
+        """Unselect the current folder and release ressources associated.
```

Comment:
typo: ressources 

also: "release associated resources" is probably flows better (swap the words).